plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -377,7 +381,7 @@ def update_client(self, id, clientrep, realm="master"):
     def create_client(self, clientrep, realm="master"):
         """ Create a client in keycloak
         :param clientrep: Client representation of client to be created. Must at least contain field clientId
-        :param realm: realm for client to be created
+        :param realm: realm for client to be created.
```

Comment:
Out of curiosity: why change this line, but not the one above, where a trailing period is also missing?

Reply:
It was an omission, I will also add it for the above line.

Reply:
Fix with commit ccd1971